src/plugins/intel_gpu/src/graph/proposal.cpp

```diff
@@ -26,7 +26,7 @@ primitive_type_id proposal::type_id() {
 }
 
 layout proposal_inst::calc_output_layout(proposal_node const& node) {
-    assert(node.get_primitive()->output_data_types.empty() &&
+    assert(static_cast<bool>(node.get_primitive()->output_data_types.at(0)) == false &&
```

Comment:
same as above, can we sure that there will be at least one element ?

Reply:
For the above reason, the condition statement has been changed as the original code( static_cast<bool>(node.get_primitive()->output_data_type) == false )